engine/src/main/java/org/entando/entando/aps/servlet/ExceptionHandlerServlet.java

```diff
@@ -0,0 +1,67 @@
+package org.entando.entando.aps.servlet;
+
+import com.agiletec.aps.system.services.lang.ILangManager;
+import com.agiletec.aps.system.services.lang.Lang;
+import com.agiletec.aps.system.services.page.IPage;
+import com.agiletec.aps.system.services.page.IPageManager;
+import com.agiletec.aps.system.services.url.IURLManager;
+import java.io.IOException;
+import java.util.Map;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+@Slf4j
+public class ExceptionHandlerServlet extends HttpServlet {
```

Comment:
@zonia3000 There is a control subservice (class com.agiletec.aps.system.services.controller.control.ErrorManager) that is delegated to do just this job in case of failure in portal-ui. Has this part been analyzed? Why it does not work?

Reply:
I wasn't aware of it. I'll take a look.